website/pages/en/developing/creating-a-subgraph.mdx

```diff
@@ -146,6 +146,8 @@ dataSources:
 
 The important entries to update for the manifest are:
 
+- `specVersion`: a semver version indicating which version of this API is being used. See [specVersion releases](#specversion-releases) section to see more details on features & releases.
```

Comment:
> a semver version indicating which version of this API is being used

This is a bit unclear, I think something like "SemVer version identifying the supported manifest structure and functionality for the subgraph. The latest version is 1.2.0".

Reply:
Updated the description.